source/_posts/2018-07-11-hassio-images.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "New Hass.io images"
+description: "We release our new images."
+date: 2018-07-11 00:00:00
+date_formatted: "April 22, 2018"
+author: Pascal Vizeli
+author_twitter: pvizeli
+comments: true
+categories: Announcements
+og_image: /images/blog/2018-07-hassio-images/blogpost.png
+---
+
+After 4 month we are happy to announce our new images for Hass.io, based on HassOS.
```

Comment:
After 4 months of ______, we are happy to announce our new images for Hass.io, based on HassOS.